### PR TITLE
Update NuGetScratch related doc

### PR DESCRIPTION
**Pull request by heng-liu**
This is fixing part of the issues in https://github.com/NuGet/docs.microsoft.com-nuget/issues/2996

### CODE REVIEW

**Review by erdembayar — APPROVED** [review]

**Comment by kartheekp-ms** [comment]
I think we also need to update https://learn.microsoft.com/en-us/nuget/reference/cli-reference/cli-ref-environment-variables doc

File: docs/consume-packages/managing-the-global-packages-and-cache-folders.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ Whenever you install, update, or restore a package, NuGet manages packages and p
...
 | http-cache | The Visual Studio Package Manager (NuGet 3.x+) and the `dotnet` tool store copies of downloaded packages in this cache (saved as `.dat` files), organized into subfolders for each package source. Packages are not expanded, and the cache has an expiration time of 30 minutes.<br/><ul><li>Windows: `%localappdata%\NuGet\v3-cache`</li><li>Mac/Linux: `~/.local/share/NuGet/v3-cache`</li><li>Override using the NUGET_HTTP_CACHE_PATH environment variable.</li></ul> |
-| temp | A folder where NuGet stores temporary files during its various operations.<br/><li>Windows: `%temp%\NuGetScratch`</li><li>Mac/Linux: `/tmp/NuGetScratch`</li></ul> |
+| temp | A folder where NuGet stores temporary files during its various operations.<br/><li>Windows: `%temp%\NuGetScratch`</li><li>Mac: `/tmp/NuGetScratch`</li><li>Linux: `/tmp/NuGetScratch<username>`</li><li>Override using the NUGET_SCRATCH environment variable.</li></ul> |
```
Comment (nkolev92) [inline docs/consume-packages/managing-the-global-packages-and-cache-folders.md:18]:
```suggestion
| temp | A folder where NuGet stores temporary files during its various operations.<br/><li>Windows: `%temp%\NuGetScratch`</li><li>Mac: `/tmp/NuGetScratch`</li><li>Linux: `/tmp/NuGetScratch/<username>`</li><li>Override using the NUGET_SCRATCH environment variable.</li></ul> |
```

Missing a slash

Comment (heng-liu) [reply docs/consume-packages/managing-the-global-packages-and-cache-folders.md:18]: We now append username to `NuGetScratch`, so there is no slash. 
E.g. for user User1, it's  `NuGetScratchUser1`.
Pls refer to code at https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Core/NuGet.Common/PathUtil/NuGetEnvironment.cs#L40-L41

Comment (nkolev92) [reply docs/consume-packages/managing-the-global-packages-and-cache-folders.md:18]: Ah, my bad then. Thanks for the link.

**Review by nkolev92 — APPROVED** [review]

**Pull request merged**